runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -511,6 +511,19 @@ public void clearCache(boolean includeDiskFiles) {
     }
 
     /**
+     * Clear the resource cache. Note that it only clear the cache for the specified
+     * url.
+     * @param url indicate which cache will be cleared.
+     * @since 6.0
+     */
+    @XWalkAPI
+    public void clearCacheForSingleFile(String url) {
```

Comment:
If there are two XWalkViews, and load the same url, Will all this url related cache being deleted?